dhall/tests/Dhall/Test/Import.hs

```diff
@@ -46,7 +47,7 @@ getTests = do
             , importDirectory </> "success/unit/asLocation/RemoteChainMissingA.dhall"
             ]
 
-    successTests <- Test.Util.discover (Turtle.chars <> "A.dhall") successTest (do
+    successTests <- Test.Util.discover (Turtle.chars <* "A.dhall") successTest (do
```

Comment:
Could you fix the glob for the `failureTests` below in the same way?

Reply:
The success tests only need this because they construct multiple filenames from the same prefix, which the failure tests don't. This same difference appears in https://github.com/dhall-lang/dhall-haskell/blob/3efdc6dcf6fe3246682fcbbf1971fca43717a65f/dhall/tests/Dhall/Test/Parser.hs#L38-L44

Is it worth making them all use `*>` for the consistency, or keep using `<>` when we don't need to construct multiple paths?

(I can see benefits to either way so I'm happy with whatever you'd prefer)

Reply:
Ah, thanks for pointing out the difference! I think it would be worth standardizing on `<*`, but it's not super important to me.